packages/polaris-viz/src/components/FunnelChart/components/Label.tsx

```diff
@@ -58,16 +61,18 @@ export function Label({
             fontSize: `${fontSize}px`,
             color,
             lineHeight: `${HORIZONTAL_BAR_LABEL_HEIGHT}px`,
-            height: HORIZONTAL_BAR_LABEL_HEIGHT,
           }}
         >
           <span
             style={{
               backgroundColor,
+              height: HORIZONTAL_BAR_LABEL_HEIGHT,
+              userSelect: 'none',
```

Comment:
`userSelect: none` is to add back the same functionality we had where elements in the chart were non-selectable